skills/csharp/experimental/eventskill/Responses/ResponsesAndTexts/ResponsesAndTexts.lg

```diff
@@ -0,0 +1,3 @@
+[import](..\FindEvents\FindEventsResponses.lg)
```

Comment:
why "FindEventResponse" not exist in other language file?

Reply:
Because there is only FindEventsResponses.lg and no other locales..

Reply:
ok..

Reply:
leave as it is.